client/datasync-client/example/README.md

```diff
@@ -0,0 +1,18 @@
+### Running example
+
+Example needs to run in browser environment.
+
+#### Steps to run example
+
+Install required packages
+
+    npm install browserify
+    npm install typescript
+    npm install tsify
+    npm install browserify-shim
+
+Build single sync bundle.
+
+    browserify --debug index.ts -p [ tsify --noImplicitAny ] > bundle.js
```

Comment:
We could add these as devDependencies and put this as a npm script. Don't think there's any harm in having extra devDependencies for a demo.